pil_analyzer/src/lib.rs

```diff
@@ -192,11 +192,13 @@ pub struct PolynomialReference {
     /// Identifier for a polynomial reference.
     /// Optional because it is filled in in a second stage of analysis.
     /// TODO make this non-optional
-    pub poly_id: Option<(u64, PolynomialType)>,
+    pub poly_id: Option<PolyID>,
     pub index: Option<u64>,
     pub next: bool,
 }
 
+pub type PolyID = (u64, PolynomialType);
```

Comment:
Would it be clearer to use an actual struct instead of an enum here? `.type` is clearer than `.1` imo.

Reply:
Maybe we should also use `ColID`?

Reply:
It's kind of mixed all over the place...